src/gluonts/mx/trainer/_base.py

```diff
@@ -147,13 +177,7 @@ def __init__(
         assert (
             0 < learning_rate < float("inf")
         ), "The value of `learning_rate` should be > 0"
-        assert (
-            0 <= learning_rate_decay_factor < 1
-        ), "The value of `learning_rate_decay_factor` should be in the [0, 1) range"
-        assert 0 <= patience, "The value of `patience` should be >= 0"
-        assert (
-            0 <= minimum_learning_rate
-        ), "The value of `minimum_learning_rate` should be >= 0"
+
```

Comment:
Minor but: maybe we can keep these assertions right after the deprecation warnings above?

Reply:
They are, aren't they?